src/functionalTest/groovy/org/danilopianini/gradle/gitsemver/functionalTest/GitSemVerFunctionalTest.groovy

```diff
@@ -0,0 +1,111 @@
+package org.danilopianini.gradle.gitsemver.functionalTest
+
+import org.danilopianini.gradle.gitsemver.functionalTest.extension.ProcessExtensions
+import org.danilopianini.gradle.gitsemver.functionalTest.extension.StringExtensions
+import org.danilopianini.gradle.gitsemver.functionalTest.specification.GradleBuildFunctionalTest
+import spock.util.mop.Use
+
+import java.util.regex.Pattern
+
+@Use([StringExtensions, ProcessExtensions])
+class GitSemVerFunctionalTest extends GradleBuildFunctionalTest {
+
+    public static final Pattern ARCHEO_TIME = ~/0\.1\.0-archeo\+[0-9]+/
```

Comment:
In groovy, `public` is not necessary because properties are public by default and groovy always generates setters and getters for properties. Calling `this.ARCHEO_TIME` internally calls `this.getARCHEO_TIME()`, so getters and setters are always called implicitly. Direct access to properties is possible by calling `this.@ARCHEO_TIME`.

Reply:
fixed, thanks